tests/Functional/SQL/PostgresNativePositionalParametersTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\SQL;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use Doctrine\DBAL\Types\Types;
+
+final class PostgresNativePositionalParametersTest extends FunctionalTestCase
+{
+    public function testPostgresNativePositionalParameters(): void
+    {
+        if (! TestUtil::isDriverOneOf('pgsql')) {
+            self::markTestSkipped('This test requires the pgsql driver.');
+        }
```

Comment:
What about the PDO driver? Does it support this style of positional parameters? The behavior of the two drivers should be as similar as possible.

Reply:
I tested with the pdo_pgsql driver and it does not seem to support this style of positional parameters. Only the native pgsql driver supports this native postgresql functionality